### ISSUE TITLE
How to map Flock and Nest teams on the Coop

### ISSUE THREAD

**Issue by 0xGabi** [aragoncoop/ideas#7]
We should have a workflow to facilitate existing teams (Flock/Nest) of the community to interact effortless with the Coop. 

For example, to allocate funds to specific issues and let the Coop manage contributors and curate the issues.

**Comment by sepu85** [comment]
this seems to be a good idea to include within the Coop scope, but isn't the AA specifically accountable for the task you are mentioning? maybe louisgrx can comment on this...

**Comment by LouisGrx** [comment]
Hey there, amazing idea. So this would mean that Nest (and Flock) teams can request help from the Coop to create and manage a bounty in the CFDAO?

In my very subjective opinion this sounds great and definitely one of the most important thing to do at the Coop level. I know you guys have plenty of projects already atm so maybe we should see where this can fit and what the process is. 

I don't know if we want a particular flow other than: 
- a repo/other for Nest teams to submit requests and engage the conversation with coop workers/community.
- Then ideally from there someone from the Coop, could take care of proposal creation on CFDAO (which can still be rejected by CFDAO voters) etc... 
- Then maybe determine a duration after which the bounty is closed if Coop cannot find anyone to fulfill it. 
- Ensure update on bounty status is provided to Nest team during the process.
- Maybe Nest teams should have the last word on the person selected to undertake the bounty and evaluating the final output.

Happy to hear your thoughts

**Comment by 0xGabi** [comment]
Yes @LouisGrx this is exactly what I would like to see the Coop accomplish. Maybe at first tried to make it as easy as possible, but on the long term it would be great to have granular roles to decide if a contributor is accepted by the teams requesting help, etc.

The main goals I can think to make this happen are:
1. Help teams curate issues they want to delegate
2. Be the gateway for new contributors to be onboard on Aragon

**Comment by yeqbfgxjiq** [comment]
The main goals I can think to make this happen are:
    - Help teams curate issues they want to delegate
    - Be the gateway for new contributors to be onboard on Aragon

@0xGabi Cool! That sounds awesome :)

@lkngtn outlined a lot of that stuff in the [Coop Workflow Proposal](https://hackmd.io/pUuLOvlQQoygsaNv0hQ8Jg?view) and that I was trying to make it more tangible with [The Coop](https://github.com/burrrata/the-coop). I stopped working on it because I wasn't sure if that's something the broader community was on board with, but it if there's interest in moving that direction then happy to do so. The Coop [website](https://www.burrrata.ch/the-coop/) is currently geared towards dev onboarding/support, but the [Framework Docs](https://www.burrrata.ch/the-coop/docs/framework/intro) can generalize (or be modified) to what was described in this thread. Also the UI is super easy to change around. If you want to help turn the whole thing into a more formal proposal I'd be happy to collaborate and present it to the group whenever they're ready.

**Comment by julianbrooks** [comment]
Proposal folded into the Coop's _Community Fund DAO (CFDAO) Working Group_, current 'Working Paper & Research Notes' page. A live copy of this document may be found [here](https://hackmd.io/cOj9xdQuQimnYp58Y26Q-g?both)
